ts/components/basic/SessionRadioGroup.tsx

```diff
@@ -32,9 +32,9 @@ export const SessionRadioGroup = (props: Props) => {
   const { items, group, initialItem, style } = props;
   const [activeItem, setActiveItem] = useState('');
 
-  useEffect(() => {
+  useMount(() => {
```

Comment:
What is the benefit of useMount over useEffect in various places?

Reply:
now that we have eslint hooks checks, 
a useEffect used just as a initial setup (dependencies of []) is not allowed anymore if inside that useEffect we have state or props variable.
In the few cases I've replaced those useEffect with useMount, it's because we specifically want this action to be run once, and not be rerun everytime the props/state gets updated.

So it was either
- disable the eslint hook props for that line
- make use of the useMount

Reply:
Nice. Yeah that works for me! Actually it's nice to have the difference 😄